models/DataObject/ClassDefinition/Data/Multiselect.php

```diff
@@ -458,6 +458,14 @@ public function getFilterConditionExt($value, $operator, $params = [])
                 $key = $params['brickPrefix'].$key;
             }
 
+            if (str_contains($name, 'cskey') && is_array($value) && !empty($value)) {
+                $values = array_map(function ($val) {
+                    return "'%" . addslashes($val) . "%'";
```

Comment:
I don't think this is appropriate escaping of values that are used in a DB query.

Reply:
@msoroka How is it going about this? Please have a look at https://github.com/pimcore/pimcore/blob/11.x/models/Asset/Dao.php#L204 And could you also please merge 10.5 in your branch to fix the tests? Thanks

Reply:
@jdreesen @robertSt7 Thanks for suggestions